### PR TITLE
Bugfix in Develop branch

### PR DESCRIPTION
**Pull request by ronzeiller**
I think there is/was a bug in the rxValidEvent and rxErrorEvent Counter in NMEAService.cpp

.event --> counter++
.countEvent --> get the count value

### CODE REVIEW

File: src/host/services/NMEAService.cpp [hunk 1]
```diff
@@ -149,8 +150,9 @@ void NMEAService::loop() {
   DEBUG("Found %i sentences waiting", receiveQueue.size());
   for (LinkedList<NMEASentence>::iterator it = receiveQueue.begin(); it != receiveQueue.end(); it++) {
     if (it->isValid()) {
-      KBoxMetrics.countEvent(rxValidEvent);
-      this->sendMessage(*it);
+      KBoxMetrics.event(rxValidEvent);
+      // DEBUG( "Rx NMEAValidCounter: %i", KBoxMetrics.countEvent(rxValidEvent) );
```
Comment (sarfata) [inline src/host/services/NMEAService.cpp]: Can you remove this line? WE are not going to need it.

File: lib/readme.txt [hunk 2]
```diff
@@ -0,0 +1,36 @@
+
```
Comment (sarfata) [inline lib/readme.txt]: I don't think we should add this file. There is already a readme in the lib folder which has more important and more relevant content.

File: src/host/services/NMEAService.cpp (hunk 1 above)
Comment (ronzeiller) [reply src/host/services/NMEAService.cpp]: debug message is deleted.
the readme.txt was made by platformio automatically (always restored when I deleted it)
I will have to find a way to experiment and try, including debug messages for testing without "damaging" your code....

Comment (sarfata) [reply src/host/services/NMEAService.cpp]: It looks like you did not commit the change to remove that debug message. Can you remove it?

**Comment by sarfata** [comment]
Thanks @ronzeiller - This is now merged. We should use the same system for every new feature but I will help you.

**Pull request merged**